knowledge/example_test.go

```diff
@@ -56,15 +57,20 @@ func ExampleClient_Read() {
 		_ = client.Close()
 	}()
 
-	path := "(n:Person)-[:HAS]->(s:Store)"
-	conditions := "WHERE n.external_id = $external_id"
+	query := "(n:Person)-[:HAS]->(s:Store) WHERE n.external_id = $external_id"
```

Comment:
you need to include `MATCH` at the start of the query

Reply:
oops